Vagrantfile

```diff
@@ -7,8 +7,21 @@ Vagrant.configure("2") do |config|
   # please see the online documentation at vagrantup.com.
 
   # Every Vagrant virtual environment requires a box to build off of.
-  config.vm.box = "ubuntu/xenial64"
+  config.vm.box = "ubuntu/trusty64"
   config.vm.provision :shell, :path => "scripts/bootstrap.sh"
+  config.vm.provision "shell", inline: <<-SHELL
```

Comment:
Why are we running this as shell within Vagrant rather than in `scripts/ol-start.sh`?